pkg/controller/perconaservermongodbbackup/backup.go

```diff
@@ -177,9 +178,24 @@ func (b *Backup) Status(ctx context.Context, cr *api.PerconaServerMongoDBBackup)
 	}
 	status.PBMPod = node
 
+	meta, err = b.pbm.GetBackupMeta(ctx, cr.Status.PBMname)
+	if err != nil || meta == nil || meta.Replsets == nil {
+		return status, nil
+	}
+
+	status.PBMPods = backupPods(meta.Replsets)
+
 	return status, nil
 }
 
+func backupPods(replsets []pbmBackup.BackupReplset) map[string]string {
+	pods := make(map[string]string)
+	for _, rs := range replsets {
+		pods[rs.Name] = strings.Split(rs.Node, ".")[0]
```

Comment:
```suggestion
		spl := strings.Split(rs.Node, ".")
		if len(spl) == 0 {
			continue
		}
		pods[rs.Name] = spl[0]
```

I think it's better to have a check here, just in case

Reply:
Agree! [29d3d12](https://github.com/percona/percona-server-mongodb-operator/pull/1500/commits/29d3d12317d4c66d2c8675db84163255bc7da7d1)